attributes/default.rb

```diff
@@ -0,0 +1,6 @@
+
+ 
+default['export-node']['path'] = ENV['TEMP']
+default['export-node']['path'] = '/opt/kitchen/export-node' unless node['platform_family'] == 'windows'
```

Comment:
I'm not agree, modern linux distro has a systemd and it going to clean /tmp files from time to time. For example, inspec tests may rely on exported file so they would fail if the file with node attributes was deleted